installing/installing_rhv/installing-rhv-default.adoc

```diff
@@ -9,11 +9,15 @@ toc::[]
 ====
 Due to a known issue, this default installation procedure does not work with {product-title} versions 4.4 and 4.5 on {rh-virtualization-first} 4.4.1.  This defect is fixed in {rh-virtualization} 4.4.2.
 
-Instead, follow the steps in link:https://docs.openshift.com/container-platform/4.5/installing/installing_rhv/installing-rhv-customizations.html[Installing a cluster on RHV with customizations].
+Instead, follow the steps in link:https://docs.openshift.com/container-platform/4.5/installing/installing_rhv/installing-rhv-customizations.html[Installing a cluster on {rh-virtualization} with customizations].
```

Comment:
Helping users notice the critical information in the second paragraph takes priority over reducing paragraphs. The two paragraphs contain related but distinct ideas. The position of "instead" establishes a linking juxtaposition. The proposed changes create a single block of text that is less likely to produce the desired outcome than the original.

I removed the third paragraph because the information it provides isn't needed in this topic.